src/components/Footer.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+import {
+  Collapse,
+  Navbar,
+  NavbarToggler,
+  NavbarBrand,
+  Nav,
+  NavItem,
+  NavLink,
+  NavbarText,
+} from "reactstrap";
+
+class Footer extends Component{
+  render(){
+    return(
+      <div>
+      <Navbar color="light" expand="md" fixed="bottom" light>
+        <NavbarBrand href="/">Thanks for trying our app!1</NavbarBrand>
+        <NavbarToggler onClick={function noRefCheck() {}} />
```

Comment:
This seems to be referencing a function that doesn't exist. It may be work cleaning this up.

Reply:
Will investigate today!